gcovr/tests/cascaded-nonsort/Makefile

```diff
@@ -0,0 +1,34 @@
+CFLAGS= -fprofile-arcs -ftest-coverage -fPIC
+
+all: copy
+	$(CXX) $(CFLAGS) -c subdir/A/file1.cpp -o subdir/A/file1.o
+	$(CXX) $(CFLAGS) -c subdir/A/file2.cpp -o subdir/A/file2.o
+	$(CXX) $(CFLAGS) -c subdir/A/file3.cpp -o subdir/A/file3.o
+	$(CXX) $(CFLAGS) -c subdir/A/file4.cpp -o subdir/A/file4.o
+	$(CXX) $(CFLAGS) -c subdir/A/file7.cpp -o subdir/A/file7.o
+	$(CXX) $(CFLAGS) -c subdir/A/C/file5.cpp -o subdir/A/C/file5.o
+	$(CXX) $(CFLAGS) -c subdir/A/C/D/file6.cpp -o subdir/A/C/D/file6.o
+	$(CXX) $(CFLAGS) -c subdir/B/main.cpp -o subdir/B/main.o
+	$(CXX) $(CFLAGS) subdir/A/file1.o subdir/A/file2.o subdir/A/file3.o subdir/A/file4.o subdir/A/C/file5.o subdir/A/C/D/file6.o subdir/A/file7.o subdir/B/main.o -o subdir/testcase
+
+run: html
+
+coverage.json:
+	./subdir/testcase
+	$(GCOVR) -r subdir --json-pretty --json $@
+
+html: coverage.json
+	$(GCOVR) -r subdir -a $< --html-cascaded-directories --html-details -o coverage.html
+
+clean:
+	rm -rf ./subdir
+	rm -f *.gc* */*.gc* */*/*.gc* */*/*/*.gc* */*/*/*/*.gc*
+	rm -f *.o */*.o */*/*.o */*/*/*.o */*/*/*/*.o
+	rm -f coverage*.*
+
+copy:
+ifeq ($(filter $(BASE_OS),MSYS_NT MINGW64_NT),)
+	cp -fr ../nested/subdir .
+else
+	COPY /Y ../nested/subdir subdirs
```

Comment:
These lines are not running windows-style in the unit tests.

Reply:
Sorry, I don't understand this.

Reply:
This is tested now to be working correctly.